go-controller/pkg/libovsdb/ops/chassis.go

```diff
@@ -151,10 +151,10 @@ func CreateOrUpdateChassis(sbClient libovsdbclient.Client, chassis *sbdb.Chassis
 			BulkOp:         false,
 		},
 		{
-			Model:          chassis,
-			OnModelUpdates: onModelUpdatesAllNonDefault(),
-			ErrNotFound:    false,
-			BulkOp:         false,
+			Model:            chassis,
+			OnModelMutations: []interface{}{&chassis.OtherConfig},
```

Comment:
this means only the key:val we provided will be inserted into the existing map without touching other existing stuff right?

Reply:
yes, Surya. Mutate insert is going to behave that way.